tasks/install.yml

```diff
@@ -88,4 +94,6 @@
     state: present
   when:
     - restic_user != 'root'
+    - restic_group != 'root'
     - not ansible_check_mode
```

Comment:
@TheLastProject This does not come from my PR, but still: what is this line for? The tasks are always run when _not ansible_check_mode_, no? Did I make a mistake in my thinking?

Reply:
.. and skipped when in Check/Test-Mode.

Reply:
I think it's just to not fail if the capabilities are not set